airbyte-scheduler/src/main/java/io/airbyte/scheduler/temporal/TemporalClient.java

```diff
@@ -43,12 +43,13 @@ public TemporalClient(WorkflowClient client) {
     this.client = client;
   }
 
-  public ConnectorSpecification submitGetSpec(long jobId, int attempt, JobGetSpecConfig config) {
+  public OutputAndStatus<JobOutput> submitGetSpec(long jobId, int attempt, JobGetSpecConfig config) {
```

Comment:
as part of subsequent refactor we will go back to just returning the naked object (e.g. `ConnectorSpecification`) instead of the JobOutput and OutputStatus wrappers. But will be easier to do this in one go after everything is moved.